src/chatlog/chatmessage.cpp

```diff
@@ -184,8 +184,11 @@ QString ChatMessage::detectAnchors(const QString &str)
                 "(?:\\b)(www\\.|http[s]?|ftp)://" // (protocol)://(printable - non-special character)
                 // http://ONEORMOREALHPA-DIGIT
                 "\\w+\\S+)" // any other character, lets domains and other
-                "|(^tox:[@\\w]+$)"); // starts with `tox` then : and only alpha-digits till the end
-                // also accepts tox:agilob@net as simplified TOX ID
+                "|"   // standard protocols OR tox ID:
+                "^tox:(([\\w]+@[\\S]+)" // tox:something@something.com
+                "|(\\w{76}))$" // tox:76 characters long tox id
```

Comment:
It must be 76 long, not from 1 to 76 `\\x` is for UTF8 characters, not standard ASCII.

btw. `\x` is invalid as it tries to escape `x` character. Alternatively it can be written as [A-Fa-f0-9]{76} but it's not for better performance.